### ISSUE TITLE
Lyric offset button screen reader speech is not defined

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#31740]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

With NVDA running go to the format menu and open the style dialogue box.  Select the Lyric grouping from the 
categories list. Tab to the lyric Text grouping area and as you arrow on to the Lyric Offset spin buttons you hear "lyric spin button", this is the same for each making it impossible to tell the difference between the two or know what they are.  They should announce something like "lyric offset below spin button editable", "lyric offset above spin button editable".

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
